frontend/ui-picasso/app/components/Organisms/CrowdloanRewards/ClaimloanPage.tsx

```diff
@@ -162,6 +162,8 @@ export const ClaimloanPage = ({ isStable = false }: Claimloan) => {
     } else {
       appDispatch(setUseAssociationMode({ useAssociationMode: "ethereum" }));
     }
+    // Only to be called on page load therefore we can omit dependencies.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
The same - We could remove the dependency list instead

Reply:
in that case, i'm getting a warning that there isn't any dependency list provided.